apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/builder/tests/ApiTestingEnvironment.java

```diff
@@ -507,6 +511,22 @@ protected IApiBaseline getWorkspaceProfile() {
 		return ApiPlugin.getDefault().getApiBaselineManager().getWorkspaceBaseline();
 	}
 
+	public static void dispose(IApiBaseline apiBaseline) {
+		if (apiBaseline instanceof WorkspaceBaseline) {
+			try {
+				IApiBaselineManager apiBaselineManager = ApiPlugin.getDefault().getApiBaselineManager();
+				Method disposeWorkspaceBaseline = ApiBaselineManager.class
+						.getDeclaredMethod("disposeWorkspaceBaseline"); //$NON-NLS-1$
```

Comment:
Why use reflection? Make the method public. I guess that's not a big deal, the manager is not API anyway

Reply:
Done.